packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -162,6 +179,22 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
       );
     }
   }
+  private _editorOnChange = (editor: ITextModel) => {
+    import('@uifabric/tsx-editor').then(editorModule => {
```

Comment:
Probably instead of re-importing this every onChange, you should probably do the import in `_onToggleCodeClick` and save the `editorModule` in a private member.